recipe-finder-fe/src/Pages/NotFound/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Header from "../../Components/Header";
+
+function NotFound() {
+  return (
+    <>
+      <Header title="FoodHub" />
+      <div className="text-center flex flex-col items-end flex-grow justify-center bg-[url('../public/img/icecream.jpg')] bg-cover bg-center bg-no-repeat">
+        <div className="w-6/12 text-center mr-4 lg:mr-20">
+          <h1 className="text-3xl lg:text-6xl font-semibold">404</h1>
+          <h2 className="text-xl lg:text-3xl">Page not found</h2>
+          <p className="text-md lg:text-xl">
+            Try going back to the previous page, or go home{" "}
```

Comment:
You can use a `Link` component here to provide a way to click to go home.

Reply:
Done